htdocs/core/modules/commande/doc/pdf_einstein.modules.php

```diff
@@ -372,6 +372,12 @@ public function write_file($object, $outputlangs, $srctemplatepath = '', $hidede
 						if (!empty($salerepobj->signature)) $notetoshow = dol_concatdesc($notetoshow, $salerepobj->signature);
 					}
 				}
+                // Extrafields in note
```

Comment:
The template pdf_einstein.modules.php , crabe and azur will be abandonned and replaced with cyan, eratosthen, ...
So can you also publish this in the new templates so we won't loose this when we will remove the file einstein, crabe, azur ?

Reply:
@eldy I took that from the new templates